### PR TITLE
Do list dodać i18next

### PR DESCRIPTION
**Pull request by tomaszprc**

### CODE REVIEW

File: src/containers/ServiceDetails/ServiceDetails.tsx [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 import styled from "styled-components";
+import { useTranslation } from "react-i18next";
 
 interface OwnProps {}
 
 type Props = OwnProps;
 
 const ServiceDetails: FC<Props> = () => {
+
+    const { t } = useTranslation(["tile"]);
```
Comment (Andrzej-Marek) [inline src/containers/ServiceDetails/ServiceDetails.tsx:12]: 1) you don't have to use array here.
2) Name it different, no tile. Make json something like carSpecification

File: src/locales/pl/index.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import common from "./common.json";
+import tile from "./tile.json";
 
 export default {
     common,
+    tile
```
Comment (Andrzej-Marek) [inline src/locales/pl/index.js:6]: carSpecification

File: src/components/Lists/CarDetailsList/CarDetailsList.tsx [hunk 3]
```diff
@@ -2,25 +2,29 @@ import { media } from "@/shared/utils";
...
 import { ListElement } from "../components";
+import { useTranslation } from "react-i18next";
 
 interface OwnProps {}
 
 type Props = OwnProps;
 
 const CarDetailsList: FC<Props> = () => {
+
+    const { t } = useTranslation(["tile"]);
```
Comment (Andrzej-Marek) [inline src/components/Lists/CarDetailsList/CarDetailsList.tsx:13]: Why you're using array here? 
2) Make JSON that have a better name like carSpecification

**Review by Andrzej-Marek — CHANGES_REQUESTED** [review]

**Comment by tomaszprc** [comment]
Fixed

**Review by Andrzej-Marek — CHANGES_REQUESTED** [review]

File: src/components/Lists/CarDetailsList/CarDetailsList.tsx [hunk 4]
```diff
@@ -2,25 +2,29 @@ import { media } from "@/shared/utils";
...
+    const { t } = useTranslation("carSpecification");
+
     return (
         <ListWrapper>
             <Column>
-                <ListElement label="Marka" value="Mercedes Benz" />
-                <ListElement label="Model" value="Mercedes Benz" />
-                <ListElement label="Rok produkcji" value="Mercedes Benz" />
-                <ListElement label="Przebieg" value="Mercedes Benz" />
+                <ListElement label={t("carSpecification:carDetails.mark")} value="Mercedes Benz" />
```
[the remark this answers is not in the document]
Comment (tomaszprc) [reply src/components/Lists/CarDetailsList/CarDetailsList.tsx]: fixed

**Review by Andrzej-Marek — APPROVED** [review]

**Pull request merged**